kbase-extension/static/kbase/css/all_concat.css

```diff
@@ -0,0 +1,5745 @@
+/** This file is generated automatically; edits will not be saved.
```

Comment:
This file contains all the css in `static/kbase/custom` and `static/kbase/css` concatenated together.

Reply:
What is the reason for putting all of it together?

Reply:
To be able to serve a single minified file, which is much more efficient than having to make a bunch of HTTP requests for smaller files.

Reply:
removing this file, will put it into a different PR